crates/sui/src/genesis_ceremony.rs

```diff
@@ -97,7 +101,9 @@ pub fn run(cmd: Ceremony) -> Result<()> {
 
         CeremonyCommand::AddValidator {
             name,
-            key_file,
+            validator_key_file,
+            staking_key_file: _,
```

Comment:
is `_` used?

Reply:
It is not yet used, we pinned it to the same key as validator_key_file